masterlist.yaml

```diff
@@ -1675,6 +1675,14 @@ plugins:
       - crc: 0xE2ADCB31
         util: 'FNVEdit v4.0.4'
 
+  - name: '(Creature)_Mojave_Straycats.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/70220/'
+        name: 'Mojave Straycats'
+    clean:
+      - crc: 0x71BA701E
+        util: 'FNVEdit v4.0.4c'
+
```

Comment:
Please sort the mod alphabetically within the `Encounters - Neutrals` section, so before `outsidebets.esp`.